packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -318,12 +362,29 @@ export default class BranchPublish extends Command {
 }
 
 function formatUpdateTitle(
-  update: Pick<Update, 'message' | 'createdAt' | 'runtimeVersion'> & {
-    actor?: Pick<Actor, 'firstName'> | null;
-  }
+  update: Exclude<
+    Exclude<ViewBranchQuery['app'], null | undefined>['byId']['updateBranchByName'],
+    null | undefined
+  >['updates'][number]
 ): string {
   const { message, createdAt, actor, runtimeVersion } = update;
-  return `[${dateFormat(createdAt, 'mmm dd HH:MM')} by ${
-    actor?.firstName ?? 'unknown'
-  }, runtimeVersion: ${runtimeVersion}] ${message}`;
+
+  let actorName: string;
+  switch (actor?.__typename) {
+    case 'User': {
+      actorName = (actor as Pick<User, 'username' | 'id'>).username;
+      break;
+    }
+    case 'Robot': {
+      const { firstName, id } = actor as Pick<Robot, 'firstName' | 'id'>;
+      actorName = firstName ?? `robot: ${id.slice(0, 4)}...`;
+      break;
```

Comment:
This is not strictly related to the `--auto` addition. Noticed we missed a case when fixing up the typing. Curious if this will fix that robot issue @jonsamp was having?

Reply:
Yeah, this should do it! Thanks for adding this along the way